netmanager/src/views/pages/Map/OverlayMap.js

```diff
@@ -637,16 +635,20 @@ export const OverlayMap = ({ center, zoom, heatMapData, monitoringSiteData }) =>
   );
 };
 
+const ENVIRONMENT = process.env.REACT_APP_BASE_URL_V2.includes('staging')
```

Comment:
Hi @Codebmk 
What if we change it to:
if (ENV is present && ENV === "staging"){
runHeatmapFunction()
}
else {
doNotRunHeatmapFunction()
}

With this approach, we are guaranteed that it will never run in PRODUCTION regardless of any edge cases we might miss out at this stage.

Please let me know what you think.